dts/bindings/radio_coex/nordic,nrf700x-coex.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+description: |
+    This is a representation of an external radio coexistence setup that has a
+    three-pin control interface (REQUEST, GRANT, PRIORITY/DIRECTION).
```

Comment:
`three-pin control interface` 
what about `swctrl1-gpios` (line 32) it seems to be fourth one... 
The description should be rewritten. The number of pins "three-pin", "four-pin" approach can become outdated quickly.

Reply:
You are right, I did not update the dts binding description. Now it simply states it is the binding for CX with nRF700x